### PR TITLE
Hide pages in docs and algolia 

### PR DESCRIPTION
**Pull request by darwin808**
## Changelog

- hide docs pages (tours,guides ,api-reference, parsley index) 
- hide algolia api
- remove packages 
 '@uiw/react-codemirror';
'@codemirror/view';
 '@codemirror/lang-javascript';
 '@uiw/codemirror-theme-github';

### CODE REVIEW

File: src/pages/docs/parsley/tour/index.js
Comment (giseleblair) [inline src/pages/docs/parsley/tour/index.js:1]: @darwin808 
only fetch production domains not webengine stage domains
` .get(
        'https://parsleydev-dev.webengine.zesty.io/-/instant/7-c005a8-zv6024.json',
      )`

Comment (giseleblair) [reply src/pages/docs/parsley/tour/index.js:1]: why do we need to set algolia api keys here?

Comment (darwin808) [reply src/pages/docs/parsley/tour/index.js:1]: why do we need to set algolia api keys here?
- its for the algolia search bar

**Review by ardeay — CHANGES_REQUESTED** [review]
This PR comments out the problem areas. The problem areas commented out needs to be investigated. 

I upgraded the CPUs to 4 with always on CPU and 2GB more of ram. This should buy us time to solve the problem @darwin808 . If it does solve the problem it will be a temporary increase in resource until the problem is resolved. 

Please return the uncommented code and review the parsley functions for iteration leaks or timeout issues. Thank you

**Comment by darwin808** [comment]
> This PR comments out the problem areas. The problem areas commented out needs to be investigated.
> 
> I upgraded the CPUs to 4 with always on CPU and 2GB more of ram. This should buy us time to solve the problem @darwin808 . If it does solve the problem it will be a temporary increase in resource until the problem is resolved.
> 
> Please return the uncommented code and review the parsley functions for iteration leaks or timeout issues. Thank you

@ardeay can we merge this to `prod` for testing/monitoring and revert the cpu/memory to original then we can monitor it if the issue exist

also most error 504 is this pages 
image

also saw our virtual size jump +8mb 
image

**Comment by ardeay** [comment]
The size of the container is not issues, it will be how the code is implemented to create a memory leak or a timeout. Let's meet

**Comment by ardeay** [comment]
@darwin808 the issue is likely in new remote requests to get data something related to the parsley integration

**Comment by darwin808** [comment]
please see #1710

**Pull request closed without merging**